apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/BigQueryNumericTest.cs

```diff
@@ -219,15 +220,33 @@ public void ToDecimal_Overflow(LossOfPrecisionHandling handling, bool negate)
         [InlineData("+.5")]
         [InlineData("1e6")]
         [InlineData("1e-6")]
-        // Other culture formatting
-        [InlineData("1234,567")]
-        [InlineData("1.234,567")]
         public void TryParse_Invalid(string text)
         {
             Assert.False(BigQueryNumeric.TryParse(text, out var value));
             Assert.Equal(BigQueryNumeric.Zero, value);
         }
 
+#if !NETCOREAPP1_0
+        [Theory]
+        [InlineData("1234,567", "es-ES")]
+        [InlineData("1.234,567", "es-ES")]
+        [InlineData("1234,567", "fr-FR")]
+        public void TryParse_UnsupportedCultures(string text, string culture)
+        {
+            CultureInfo oldCulture = Thread.CurrentThread.CurrentCulture;
+            Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture(culture);
```

Comment:
Please use try/finally so that if the test fails, we still reset the culture appropriately.

Reply:
Done.